rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerEventReceiver.java

```diff
@@ -123,7 +136,11 @@ public void on(Object t) {
 
     public void stop() {
         if (socket != null) {
-            socket.close();
+            try {
+                socket.close();
+            } catch (Exception ignored) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)